src/test/java/org/springframework/data/couchbase/domain/AirportRepository.java

```diff
@@ -39,6 +42,7 @@
 	Iterable<Airport> findAll();
 
 	@Override
+	@ScanConsistency(query = QueryScanConsistency.REQUEST_PLUS)
 	Airport save(Airport airport);
```

Comment:
isn't that a KV method which does not need the scan consistency?

Reply:
fixed